Menus/MainMenu/MainMenu.cs

```diff
@@ -121,6 +109,25 @@ void Start()
 			UpdateToolButtons();
 		}
 
+		public void SetVisible(bool visible, bool temporary = false)
+		{
+			if (m_Visible != visible)
+			{
+				m_Visible = visible;
+				if (m_MainMenuUI)
+				{
+					m_MainMenuUI.visible = visible;
+					if (!temporary)
+						SendVisibilityPulse();
+				}
+			}
+		}
+
+		public bool GetVisible()
```

Comment:
This came about because I don't like such a specific implementation of haptic feedback making its way into the menu interface. 
Capturing the 'temporaryness' is important for later, however, especially if we start alpha-ing out menus instead of hiding them completely.
Another option is this doesn't take a bool, but some kind of flags enum that can encapsulate whether this is a major or minor(temporary) operation.